serving/samples/README.md

```diff
@@ -15,6 +15,7 @@ to learn more about Knative Serving resources.
 | Github Webhook | A simple webhook handler that demonstrates interacting with Github. | [Go](gitwebhook-go/README.md) |
 | gRPC | A simple gRPC server. | [Go](grpc-ping-go/README.md) |
 | Knative Routing | An example of mapping multiple Knative services to different paths under a single domain name using the Istio VirtualService concept. | [Go](knative-routing-go/README.md) |
+| Redis Guestbook | A sample that shows how to deploy Redis alongside a Knative Serving Service. | [Go](guestbook-redis-go/README.md) |
```

Comment:
"A sample that shows how to deploy..." --> "A sample that deploys..."

Reply:
Done.